hawc/apps/lit/models.py

```diff
@@ -868,6 +868,15 @@ class Reference(models.Model):
 
     BREADCRUMB_PARENT = "assessment"
 
+    def update_tags(self, tag_pks, user):
+        user_ref_tags, created = UserReferenceTag.objects.get_or_create(reference=self, user=user)
```

Comment:
can we use `update_or_create`? it would require fewer db hits instead of getting, changing, and then updating I think?